WordPress/build.gradle

```diff
@@ -46,15 +46,18 @@ android {
         vectorDrawables.useSupportLibrary = true
 
         buildConfigField "boolean", "AZTEC_EDITOR_AVAILABLE", "true"
+        buildConfigField "boolean", "LOGIN_WIZARD_STYLE_ACTIVE", "true"
     }
 
     productFlavors {
         vanilla { // used for release and beta
             buildConfigField "boolean", "AZTEC_EDITOR_AVAILABLE", "false"
+            buildConfigField "boolean", "LOGIN_WIZARD_STYLE_ACTIVE", "false"
         }
 
         zalpha { // alpha version - enable experimental features
             applicationId "org.wordpress.android"
+            buildConfigField "boolean", "LOGIN_WIZARD_STYLE_ACTIVE", "false"
```

Comment:
The idea is to have the flag "machinery" in place and have the feature merged as soon as possible via small PRs. For now it's only for the development version but will escalate as the feature implementation matures. Does that make sense?